event.go

```diff
@@ -96,6 +121,81 @@ func formatHeaders(h http.Header, redact map[string]bool) map[string]string {
 	return ret
 }
 
+func redactValues(b any, includedKeys map[string]bool) any {
+	lowerCase := regexp.MustCompile("[a-z]")
+	upperCase := regexp.MustCompile("[A-Z]")
+	numeric := regexp.MustCompile("[0-9]")
+	special := regexp.MustCompile("[^a-zA-Z0-9\\s]")
+
+	var redactInt = func(i int) int {
+		s := strconv.Itoa(i)
+		s = numeric.ReplaceAllString(s, "1")
+		i, _ = strconv.Atoi(s)
+		return i
+	}
+
+	switch b := b.(type) {
+	case map[string]any:
+		for k, v := range b {
+			if includedKeys[k] {
+				b[k] = v
+			} else {
+				b[k] = redactValues(v, includedKeys)
+			}
+		}
+		return b
+	case []any:
+		for i, v := range b {
+			b[i] = redactValues(v, includedKeys)
+		}
+		return b
+	case []uint8:
```

Comment:
nitpick: this type is typically spelled `[]byte` (but a byte and a uint8 are the same type under the hood).

Reply:
Yeah, makes sense... I had made this by spitting out the TypeOf when I was debugging the binary test